tests/pytests/functional/modules/state/requisites/test_require.py

```diff
@@ -4,9 +4,7 @@
 
 from . import normalize_ret
 
-pytestmark = [
-    pytest.mark.windows_whitelisted,
-]
+pytestmark = [pytest.mark.windows_whitelisted, pytest.mark.core_test]
```

Comment:
```suggestion
pytestmark = [pytest.mark.windows_whitelisted, pytest.mark.core_test,]
```
Instead of marking all of these, if there's more than one entry in `pytestmark`, make sure the last entry has a `,` so that black can properly format these lists vertically.

Reply:
pylint formats it that way

Reply:
https://github.com/cmcmarrow/salt/pull/10#discussion_r1154336645

Reply:
if you add the trailing comma, it will render it vertically, and fyi, it's black, not pylint.